platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Submission.java

```diff
@@ -10,7 +10,9 @@
  */
 @Entity
 @Table(name = "submission")
-public class Submission extends IdentifiableEntity {
+public class Submission extends IdentifiableEntity implements StoredObject{
```

Comment:
this is now an persistable database entity and a stored object, i am not sure if this makes sense